lib/awspec/helper/finder/efs.rb

```diff
@@ -0,0 +1,62 @@
+module Awspec::Helper
+  module Finder
+    module Efs
+      def find_efs(id)
+        begin
+          res = efs_client.describe_file_systems({
+                                                   file_system_id: id,
+                                                   max_items: 1
+                                                 })
+        rescue
+          # Aws::EFS::Errors::BadRequest (invalid file system ID: my-efs)
+          file_system_id = get_id_by_name_tag(id)
+          res = efs_client.describe_file_systems({
+                                                   file_system_id: file_system_id,
+                                                   max_items: 1
+                                                 })
+        end
+        res.file_systems.first if res
```

Comment:
Okay, I was confused since it wasn't a ruby built in and I was getting errors when using irb. 
Sorry, I don't usually write Ruby.